From foo@baz Thu May  4 16:05:52 PDT 2017
Date: Thu, 04 May 2017 16:05:52 -0700
To: Greg KH <gregkh@linuxfoundation.org>
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Subject: scsi: advansys.c: remove DMA build warning message

From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

The driver was finally converted to the "new" DMA api in newer kernels,
but here in 3.18, it never will happen, so just shut the warning up for
now as obviously no one is going to do anything about it.

Cc: Arnd Bergmann <arnd@arndb.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

---
 drivers/scsi/advansys.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/scsi/advansys.c
+++ b/drivers/scsi/advansys.c
@@ -68,7 +68,7 @@
  *  7. advansys_info is not safe against multiple simultaneous callers
  *  8. Add module_param to override ISA/VLB ioport array
  */
-#warning this driver is still not properly converted to the DMA API
+/*#warning this driver is still not properly converted to the DMA API */
 
 /* Enable driver /proc statistics. */
 #define ADVANSYS_STATS
